mappings/net/minecraft/world/gen/noise/NoiseRouterData.mapping

```diff
@@ -0,0 +1,63 @@
+CLASS net/minecraft/unmapped/C_ronqvany net/minecraft/world/gen/noise/NoiseRouterData
+	FIELD f_craftepc VEIN_FREQUENCY D
+	FIELD f_hfbhqfdm ORE_THICKNESS F
+	FIELD f_ljabpota BLENDING_JAGGEDNESS Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_skugmypg BLENDING_FACTOR Lnet/minecraft/unmapped/C_kzwapyud;
+	FIELD f_sywrklcc NOODLE_SPACING_AND_STRAIGHTNESS D
+	FIELD f_tqarabvz SURFACE_DENSITY_THRESHOLD D
+	METHOD m_bxikrtou noodleCave ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_dqlsbhbi finalDensity (Lnet/minecraft/unmapped/C_ulvgadxi;ZZZ)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 config
+	METHOD m_duxzadwj spaghettiCaveRoughnessFunction ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_dwhlqlwh getNoise (Lnet/minecraft/unmapped/C_xhhleach;)Lnet/minecraft/unmapped/C_cjzoxshv;
+		ARG 0 key
+	METHOD m_eaewznor createEnd (Lnet/minecraft/unmapped/C_ulvgadxi;)Lnet/minecraft/unmapped/C_isyxixvk;
+		ARG 0 config
+	METHOD m_eaqwqgkd createNoiseRouter (Lnet/minecraft/unmapped/C_ulvgadxi;JLnet/minecraft/unmapped/C_tqxyjqsk;Lnet/minecraft/unmapped/C_zorcemxg$C_ybxxvwvk;Lnet/minecraft/unmapped/C_isyxixvk;)Lnet/minecraft/unmapped/C_fpiyqmft;
+		ARG 0 config
+		ARG 1 seed
+		ARG 3 noiseRegistry
+		ARG 4 random
+		ARG 5 router
+	METHOD m_gkzjvtdp yLimitedInterpolatable (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;III)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+		ARG 1 whenInRange
+		ARG 2 minInclusive
+		ARG 3 maxInclusive
+		ARG 4 maxRange
+	METHOD m_hfwlayyo createOverworldWithNoCaves (Lnet/minecraft/unmapped/C_ulvgadxi;)Lnet/minecraft/unmapped/C_isyxixvk;
```

Comment:
```suggestion
	METHOD m_hfwlayyo createOverworldWithoutCaves (Lnet/minecraft/unmapped/C_ulvgadxi;)Lnet/minecraft/unmapped/C_isyxixvk;
```
A bit of a nitpick, but I think "without" is a more common and readable phrasing than "with no".